programs/eosioc/help_text.cpp

```diff
@@ -96,9 +96,122 @@ auto smatch_to_variant(const std::smatch& smatch) {
+                   "  \"actions\":[{ \n"
+                   "    \"account\":\"account_name\",\n"
+                   "    \"name\":\"action_name\",\n"
+                   "    \"authorization\":[{ \"actor\":\"account_name\",\"permission\":\"permission_name\" }],\n"
+                   "    \"data\":\"bytes\"\n"
+                   "  }]\n"
+                   "}"
+                   "e.g.\n"
+                   "{\n"
+                   "  \"ref_block_num\":\"1000\",\n"
+                   "  \"ref_block_prefix\":\"3463702842\",\n"
+                   "  \"expiration\":\"2018-01-23T01:51:05\",\n"
+                   "  \"region\": \"0\",\n"
+                   "  \"read_scope\":[ \"initb\", \"initc\" ],\n"
+                   "  \"write_scope\":[ \"initb\", \"initc\" ],\n"
+                   "  \"actions\":[{ \n"
+                   "    \"account\":\"eosio\",\n"
+                   "    \"name\":\"transfer\",\n"
+                   "    \"authorization\":[{ \"actor\":\"initb\",\"permission\":\"active\" }],\n"
+                   "    \"data\":\"000000008093dd74000000000094dd74e80300000000000000\"\n"
+                   "  }]\n"
+                   "}"},
+        { 3120006, "Ensure that your abi JSON follows the following format!\n"
+                   "{\n"
+                   "  \"types\" : [{ \"new_type_name\":\"type_name\", \"type\":\"type_name\" }],\n"
+                   "  \"structs\" : [{ \"name\":\"type_name\", \"base\":\"type_name\", \"fields\": [{ \"name\":\"field_name\", \"type\": \"type_name\" }] }],\n"
+                   "  \"actions\" : [{ \"name\":\"action_name\",\"type\":\"type_name\"}],\n"
+                   "  \"tables\" : [{\n"
+                   "    \"name\":\"table_name\",\n"
+                   "    \"index_type\":\"type_name\",\n"
+                   "    \"key_names\":[ \"field_name\" ],\n"
+                   "    \"key_types\":[ \"type_name\" ],\n"
+                   "    \"type\":\"type_name\" "
+                   "  }]\n"
+                   "}\n"
+                   "e.g.\n"
+                   "{\n"
+                   "  \"types\" : [{ \"new_type_name\":\"account_name\", \"type\":\"name\" }],\n"
+                   "  \"structs\" : [\n"
+                   "    { \"name\":\"foo\", \"base\":\"\", \"fields\": [{ \"name\":\"by\", \"type\": \"account_name\" }] },\n "
+                   "    { \"name\":\"foobar\", \"base\":\"\", \"fields\": [{ \"name\":\"by\", \"type\": \"account_name\" }] }\n"
+                   "  ],\n"
+                   "  \"actions\" : [{ \"name\":\"foo\",\"type\":\"foo\"}],\n"
+                   "  \"tables\" : [{\n"
+                   "    \"name\":\"foobar_table\",\n"
+                   "    \"index_type\":\"i64\",\n"
+                   "    \"key_names\":[ \"by\" ],\n"
+                   "    \"key_types\":[ \"account_name\" ],\n"
+                   "    \"type\":\"foobar\" "
+                   "  }]\n"
+                   "}"}
```

Comment:
These would be much easier to read and maintain if you used: 
```
R"=====(
{
  ...
})=====";
```

Reply:
Good point. Thanks for the advice! PR Updated